internal/node/node.bzl

```diff
@@ -197,6 +204,15 @@ _NODEJS_EXECUTABLE_ATTRS = {
         """,
         default = [],
     ),
+    "jasmine_config": attr.label(
+        doc = """The path to the json configuration file of jasmine.
```

Comment:
instead of an example, should link to their docs

Reply:
Sorry, I wasn't aware of this requirement.
I will look harder at CONTRIBUTING.md (if any) next time